content/en/blog/_posts/2023-03-08-Updates-to-the-Auto-refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: blog
+title: Updates to the Auto-refreshing Official CVE Feed
+date: 2023-03-22
+slug: k8s-cve-feed-beta
+---
+
+**Authors**: Cailyn Edwards (Shopify), Pushkar Joglekar (Credit Karma), Mahé Tardy (Isovalent)
```

Comment:
```suggestion
**Authors**: Cailyn Edwards (Shopify), Mahé Tardy (Isovalent), Pushkar Joglekar
```

I am working in personal capacity so do not want to have employer name here.

Reply:
Switched the sequence to make it _alphabetical_

Reply:
ah! I made it alphabetical on the last name